src/main/java/hbv501g/recipes/Controllers/UserController.java

```diff
@@ -102,4 +112,47 @@ public User signup(HttpSession session, @RequestParam String username, @RequestP
         return null;
     }
 
+    /**
+     * Endpoint to get the pantry of a user with the specified id
+     * 
+     * @param id - user id
+     * @return
+     */
+    @GetMapping("/user/id/{id}/pantry")
+    public List<IngredientMeasurement> getUserPantry(@PathVariable(value = "id") long id) {
+        return userService.findUserPantry(id);
+    }
+
+    /**
+     * Deletes a pantry item for specified user, where the pantry item uses the
+     * specified ingredient. Assumes only one item for each ingredient
+     * 
+     * @param uid id of user owning pantry
+     * @param iid id of ingredient in pantry item to delete
+     */
+    @RequestMapping(value = "/user/pantry/delete", method = { RequestMethod.GET, RequestMethod.PUT })
```

Comment:
Á ekki að nota RequestMethod.DELETE? Var sjálf ekki búin að skoða það

Reply:
Delete er ef ég eyði notandanum, hér er bara verið að breyta attributes